package.json

```diff
@@ -46,6 +46,28 @@
     "test": "./node_modules/.bin/karma start ./karma.conf.js",
     "travis": "./node_modules/grunt-cli/bin/grunt ci --verbose --force & npm run test"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
I agree that we should be using fixed version numbers so people's local builds don't break.